adhocracy4/comments_async/static/comments_async/comment_list.jsx

```diff
@@ -14,7 +14,7 @@ const CommentList = (props) => {
               user_name={comment.user_name}
               user_pk={comment.user_pk}
               user_profile_url={comment.user_profile_url}
-              user_image={comment.user_image}
+              user_image={comment.user_image ? comment.user_image : comment.user_image_fallback}
```

Comment:
Haha, I was a bit confused initially, because this affects all projects. But right, we also have the react comments in a4. :smile: